sygnal/gcmpushkin.py

```diff
@@ -297,7 +319,9 @@ async def _request_dispatch(self, n, log, body, headers, pushkeys, span):
                 f"Unknown GCM response code {response.code}"
             )
 
-    async def _dispatch_notification_unlimited(self, n, device, context):
+    async def _dispatch_notification_unlimited(
+        self, n: Notification, device: Device, context: NotificationContext
+    ) -> Union[int, list]:
```

Comment:
```suggestion
    ) -> List[str]:
```
I don't think this can return an int — let me know if you spot something different